From: Tejun Heo <teheo@suse.de>
Subject: libata: prefer libata drivers over ide ones
References: bnc#433105

When loading modules, modprobe follows modules.order which is
generated according to the link order.  libata ones have been our
primary ATA drivers for some time now and having ide drivers before
libata ones trigger unexpected behaviors.  Prefer libata over ide.

Signed-off-by: Tejun Heo <teheo@suse.de>
---
 drivers/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/Makefile
+++ b/drivers/Makefile
@@ -39,9 +39,9 @@ obj-$(CONFIG_PARPORT)		+= parport/
 obj-y				+= base/ block/ misc/ mfd/
 obj-$(CONFIG_NUBUS)		+= nubus/
 obj-y				+= macintosh/
-obj-$(CONFIG_IDE)		+= ide/
 obj-$(CONFIG_SCSI)		+= scsi/
 obj-$(CONFIG_ATA)		+= ata/
+obj-$(CONFIG_IDE)		+= ide/
 obj-y				+= net/
 obj-$(CONFIG_ATM)		+= atm/
 obj-$(CONFIG_FUSION)		+= message/
